### PR TITLE
update hosting language and organization

### PR DESCRIPTION
**Pull request by kflemin**

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -8,14 +8,84 @@ If you are a developer, consult the [Developer Getting Started Guide](https://se
...
+
+SEED Platform&trade; is secured behind the Amazon Web Services (AWS) firewall, and other security measures are in place as well. For more information about specific security protocols, please contact us using the information found on the [contact page](contact.md).
+
+## Additional Hosting-Related Questions
+
+### Can the web interface of a self-hosted SEED instance be accessed via a local network, or is internet access required? Can access via local internet be disabled?
+
+Access via public internet would be a function of the self-hosted instance's network configuration. If SEED is hosted on Amazon Web Services, for example, the host can deny access based on CIDR blocks (IP address ranges) in the Amazon Web Services Console. If the application is hosted on user-owned infrastructure, a local firewall can prevent external access. Note that internal access in both cases would still be available (i.e., the web interface would be accessible).
+
+### What is the estimated cost to implement and maintain the SEED Platform as part of a benchmarking system?
```
Comment (nllong) [inline docs/about.md:77]:
```suggestion
### What is the estimated cost to implement and maintain the SEED Platform?
```

File: docs/about.md [hunk 2]
```diff
@@ -8,14 +8,84 @@ If you are a developer, consult the [Developer Getting Started Guide](https://se
...
+
+There are three hosting options:
+
+1. Self-hosting on user-owned hardware
+2. Self-hosting on the cloud (e.g. Amazon, Google Compute)
+3. Paying a hosting provider. Several companies offer SEED as a hosted solution:
```
Comment (nllong) [inline docs/about.md:83]: Need to add 
* DOE/National Lab-Hosted Instance

Add it to the end of the list.

**Review by nllong — CHANGES_REQUESTED** [review]
few comments... thanks for updating so quickly!

**Comment by kflemin** [comment]
@nllong I addressed your change requests and also archived the inaugural partners agreement.

File: docs/about.md (hunk 2 above)
Comment (kflemin) [reply docs/about.md:83]: done!

File: docs/about.md [hunk 3]
```diff
@@ -8,14 +8,84 @@ If you are a developer, consult the [Developer Getting Started Guide](https://se
...
+
+- Business process support to help optimize organizational data management, software integration, and data quality.
+
+- Technical Assistance with initial account set-up and load, combine, and map data sets to assemble your master buildings list; determine compliance with your program requirements; provide technical guidance and support on establishing API connections with related applications such as third-party CRM applications.
+
+- Hosting services for your SEED account.
+
+#### Data Privacy Policy
+
+You, and your company or organization (the PROVIDER) have agreed to upload and provide PROPRIETARY DATA, including energy efficiency data, building data, and building energy performance data to the Standard Energy Efficiency Data (SEED™) Platform database that the U.S. Department of Energy (the Government) and Alliance for Sustainable Energy, LLC (“Alliance”) the manager and operator of the National Renewable Energy Laboratory (“NREL”) (the RECIPIENT) have created and provide for use for registered users.
```
[the remark this answers is not in the document]
Comment (kflemin) [reply docs/about.md:37]: yep

**Review by nllong — APPROVED** [review]

**Pull request merged**